server/http/helloapp/helloapp.go

```diff
@@ -43,7 +43,7 @@ func Init(router *mux.Router, apps *apps.Service) {
 	subrouter.HandleFunc(PathWishInstall, wish(h.handleInstall)).Methods("POST")
 	subrouter.HandleFunc(PathWishConnectedInstall, wish(h.handleConnectedInstall)).Methods("POST")
 
-	subrouter.PathPrefix(PathOAuth2).HandlerFunc(h.handleOAuth).Methods(http.MethodGet, http.MethodPost)
+	subrouter.PathPrefix(PathOAuth2).HandlerFunc(h.handleOAuth).Methods("POST")
```

Comment:
Maybe I was not clear on my last comment. What we need is GET. We do NOT need POST.
Also, not sure why instead of using the constant in the http package you are just putting the string yourself.